app/controllers/posts_controller.rb

```diff
@@ -15,6 +15,19 @@ def index
     end
   end
 
+  def new; end
+
+  def create
+    data = params.require(:post).permit(:title, :text)
+    post = Post.create!(author: current_user,
+                        title: data[:title],
+                        text: data[:text],
+                        comments_counter: 0,
+                        likes_counter: 0)
+    post.update_posts_counter_for_user
+    redirect_to user_path current_user.id
+  end
+
```

Comment:
Optional -
it will be a nice addition if you added  a nice 
 [params method](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides) 
that will look somewhat like this  
`
    private 
    def post_params
      params.require(:post).permit(:title, :text)
    end
`
😄